src/oauth2-mock-server.ts

```diff
@@ -99,6 +105,10 @@ Options:
                     If omitted, 8080 will be used.
                     If 0 is provided, the operating system will assign
                     an arbitrary unused port.
+  -c <cert>         Optional file path to an SSL cert, both cert and key need
+                    to be supplied to enable SSL.
+  -k <key>          Optional file path to an SSL key, both key and cert need
```

Comment:
@lbestftr Could you slightly tweak the README to reflect this new capability?

Reply:
And while doing so, could you please hint at https://github.com/FiloSottile/mkcert as a potential solution to avoid dealing with external tooling complaining about self-signed/untrusted certificates.

Reply:
Done!